src/DynamoCore/Graph/Nodes/Attributes.cs

```diff
@@ -18,20 +18,40 @@ public NodeNameAttribute(string elementName)
         public string Name { get; set; }
     }
 
+    /// <summary>
+    ///     The NodeCategoryAttribute attribute allows the node implementer
```

Comment:
The NodeCategoryAttribute attribute allows the node implementer to define in which category the node will appear